logic/src/commonMain/kotlin/com/wire/kalium/logic/feature/call/CallManager.kt

```diff
@@ -1,16 +1,25 @@
 package com.wire.kalium.logic.feature.call
 
+import com.wire.kalium.calling.CallType
+import com.wire.kalium.calling.CallingConversationType
+import com.wire.kalium.logic.data.id.ConversationId
 import com.wire.kalium.logic.data.message.Message
 import com.wire.kalium.logic.data.message.MessageContent
 import kotlinx.coroutines.flow.StateFlow
 import kotlinx.coroutines.flow.map
 
-expect class CallManager {
-
+interface CallManager {
     suspend fun onCallingMessageReceived(message: Message, content: MessageContent.Calling)
+    suspend fun startCall(conversationId: ConversationId, callType: CallType, conversationType: CallingConversationType, isAudioCbr: Boolean = false) //TODO Audio CBR
     val allCalls: StateFlow<List<Call>>
 }
 
+expect class CallManagerImpl: CallManager {
+    override suspend fun onCallingMessageReceived(message: Message, content: MessageContent.Calling)
+    override val allCalls: StateFlow<List<Call>>
+    override suspend fun startCall(conversationId: ConversationId, callType: CallType, conversationType: CallingConversationType, isAudioCbr: Boolean)
+}
```

Comment:
No need to repeat the functions and fields here with empty overrides.

```suggestion
expect class CallManagerImpl: CallManager
```